app/api/v1/views.py

```diff
@@ -31,29 +41,27 @@ def post(self):
     def get(self):
         resp =self.db.view()
         return make_response(jsonify({
-            "Status" : 200,
-            "All Red Flags":resp
-        }),200)
+            "successfully deleted"
```

Comment:
I thought this suggestion could work in a few other scenarios that I don't necessarily need success messages.